CS483/CS483/Kartaclysm/StateMachine/Gameplay/StateRacing.cpp

```diff
@@ -100,6 +98,13 @@ void Kartaclysm::StateRacing::BeginRace()
 	HeatStroke::GameObject* pTrack = m_pGameObjectManager->CreateGameObject(m_mContextParams.at("TrackDefinitionFile"), "Track");
 	ComponentTrack* pTrackComponent = static_cast<ComponentTrack*>(pTrack->GetComponent("GOC_Track"));
 
+	// TODO: weird issue with Racer transforms unless m_vWorldTranslation is changed on walls
```

Comment:
The weird issue is that the racers overlap at the start line and can't move. I believe it has something to do with the Wall collider physics all overlapping at the start line, meaning the player is squished between invisible walls.

I suspect the issue is m_vWorldTransform, as calling GetTranslation() is the required method call to solve the issue (this is the only place in code where this variable is changed). I will investigate further, but all straightforward solutions didn't seem to work and I had spent 2.5 hours already :/